src/ReplaceTransition.tsx

```diff
@@ -47,15 +68,19 @@ class ReplaceTransition extends React.Component<Props> {
   }
 
   render() {
-    const { children, in: inProp, ...props }: any = this.props;
-    const [first, second]: any = React.Children.toArray(children);
-
-    delete props.onEnter;
-    delete props.onEntering;
-    delete props.onEntered;
-    delete props.onExit;
-    delete props.onExiting;
-    delete props.onExited;
+    const {
+      children,
+      in: inProp,
+      onEnter: _onEnter,
```

Comment:
Is the rename here only for ESLint? If so, then we should configure the rule appropriately. I remember there was an option specifically for rest-spread.

Reply:
@eps1lon Thank you! I've enabled the `ignoreRestSiblings` option at [d8dcd90](https://github.com/reactjs/react-transition-group/pull/812/commits/d8dcd90548ea035f85ede7f6fa944b4e6627fd93)